fluff/app_config.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+from django.db.models.signals import post_migrate
+from fluff.signals import catch_signal
+
+
+class FluffAppConfig(AppConfig):
+    name = 'fluff'
+
+    def ready(self):
+        post_migrate.connect(catch_signal, sender=self)
```

Comment:
Curious if you think we need to worry about [this](https://docs.djangoproject.com/en/1.8/topics/signals/#connecting-receiver-functions):
> The ready() method may be executed more than once during testing, so you may want to [guard your signals from duplication](https://docs.djangoproject.com/en/1.8/topics/signals/#preventing-duplicate-signals), especially if you’re planning to send them within tests.